contracts/src/CrossChainApplications/ERC20Bridge/ERC20Bridge.sol

```diff
@@ -73,15 +73,29 @@ contract ERC20Bridge is IERC20Bridge, ITeleporterReceiver, ReentrancyGuard {
     uint256 public constant MINT_BRIDGE_TOKENS_REQUIRED_GAS = 200_000;
     uint256 public constant TRANSFER_BRIDGE_TOKENS_REQUIRED_GAS = 300_000;
 
+    // Errors
+    error InvalidTeleporterMessengerAddress();
+    error CannotBridgeTokenWithinSameChain();
+    error InvalidRecipientAddress();
+    error InvalidDestinationBridgeAddress();
+    error InvalidBridgeTokenAddress();
+    error InsufficientTotalAmount(uint256 totalAmount, uint256 feeAmount);
+    error InsufficientAdjustedAmount(uint256 adjustedAmount, uint256 feeAmount);
+    error Unauthorized();
+    error InvalidAction();
+    error BridgeTokenAlreadyExists(address bridgeTokenAddress);
+    error InsufficientWrappedTokenBalance(uint256 currentBalance, uint256 requestAmount);
+    error CannotBridgeWrappedToken(address nativeTokenAddress);
```

Comment:
```suggestion
    // Errors
    error BridgeTokenAlreadyExists(address bridgeTokenAddress);
    error CannotBridgeTokenWithinSameChain();
    error CannotBridgeWrappedToken(address nativeTokenAddress);
    error InsufficientAdjustedAmount(uint256 adjustedAmount, uint256 feeAmount);
    error InsufficientTotalAmount(uint256 totalAmount, uint256 feeAmount);
    error InsufficientWrappedTokenBalance(uint256 currentBalance, uint256 requestAmount);
    error InvalidAction();
    error InvalidBridgeTokenAddress();
    error InvalidDestinationBridgeAddress();
    error InvalidRecipientAddress();
    error InvalidTeleporterMessengerAddress();
    error Unauthorized();
```

Super nit: Ordering these alphabetically makes them slightly easier to read through.